packages/@aws-cdk/core/test/stack.test.ts

```diff
@@ -462,6 +462,181 @@ describe('stack', () => {
     });
   });
 
+  test('cross-stack references of lists returned from Fn::GetAtt work', () => {
+    // GIVEN
+    const app = new App();
+    const stack1 = new Stack(app, 'Stack1');
+    const exportResource = new CfnResource(stack1, 'exportedResource', {
+      type: 'BLA',
+    });
+    const stack2 = new Stack(app, 'Stack2');
+    // L1s represent attribute names with `attr${attributeName}`
+    (exportResource as any).attrList = ['magic-attr-value'];
+
+    // WHEN - used in another stack
+    new CfnResource(stack2, 'SomeResource', {
+      type: 'BLA',
+      properties: {
+        Prop: exportResource.getAtt('List'),
```

Comment:
One more test: one where the customer does something like `Fn.select(3, exportResource.getAtt('List'))`, or `Fn.join('+', exportResource.getAtt('List'))`.

Reply:
Add a test against some L1 or L2 resources as well, in some downstream construct library that happens to really have a list as an attribute.